database/index.js

```diff
@@ -0,0 +1,34 @@
+const mongoose = require('mongoose');
+
+mongoose.connect('mongodb://localhost/menu', { useNewUrlParser: true });
+const db = mongoose.connection;
+db.on('error', console.error.bind(console, 'connection error:'));
+db.once('open', () => {
+  console.log('We\'re connected!');
+});
+
+const menuItemSchema = new mongoose.Schema({
+  restaurantId: Number,
+  name: String,
+  price: Number,
+  description: String,
+  category: String,
+  pictureUrl: String,
+  popular: Boolean,
+  spicy: Boolean,
+  requiredChoiceCategories: [{
```

Comment:
fyi, I didn't specific what is inside the array. I did 'requiredChoiceCategories: Array' instead to keep it simple.

Reply:
Yeah I think both ways will work. It was really useful for me to write it out because then I could remind myself what the schema is. :)